### DISCUSSION TITLE
Forking off your os

### DISCUSSION THREAD

**Discussion by dexxplay** [VendelinSlezak/BleskOS#92] [Q&A] [answered]
Can I use your os to build a diffrent os?

**Comment by VendelinSlezak** [comment, marked as answer]
Sure! That’s why I choosed MIT license. If you find code useful, grab it and use it :) Just be sure that you use MIT compatibile license.

**Comment by dexxplay** [comment]
How do I use MIT compatabile license

**Comment by VendelinSlezak** [comment]
It depends on license you choose, but basically only thing that is needed to use MIT licensed code is in MIT license specified as “The above copyright notice and this permission notice shall be included in all
copies or substantial portions of the Software.”
